lib/XML/Enc.pm

```diff
@@ -383,6 +421,8 @@ sub _setOAEPAlgorithm {
     my $self    = shift;
     my $method  = shift;
 
+    return if ! defined $method;
```

Comment:
You do an exists later on, shouldn't it either croak/die/warn about method not being specified or use the default which is used later on?

So, method having value `''` or `0` is fine, but `undef` isn't. I would perhaps do `$method //= ''`; and continue with the function. Or just `croak` if it is `undef`.